boefjes/boefjes/models.py

```diff
@@ -75,6 +77,24 @@ class Config:
         validate_assignment = True
 
 
+class BoefjeConfig(BaseModel):
+    id: int
+    settings: dict
+    enabled: bool
+    boefje_id: str
+    organisation_id: str
+
+    @computed_field  # type: ignore[prop-decorator]
+    @property
+    def env_hash(self) -> str:
+        sorted_settings = dict(
+            sorted(self.settings.items())
+        )  # settings are assumed to be a one level deep mapping of strings to strings
+        settings_bytes = json.dumps(sorted_settings).encode()
```

Comment:
why not use the sort_keys argument on dumps?
Also, if the result is a set of bytes that are deterministically sorted, just hash those without encoding them, and as a result we are not limited ourselves to any depth or types.

Reply:
Ah yes I glimpsed over the `sort_keys` option! The `encode` makes sure we have bytes, which is I believe most hash implementations expect, including those in hashlib.

Reply:
Ah true, encode can stay :p
For sort_keys, this means we are still not sorting lists present in the Json, which would mean we should move to a more deterministic string-builder.

Reply:
Yes so the assumption is that the settings are a "`dict[str, str]`", which is a safe assumption perhaps since we inject these values in the runtime environment, turning it roughly into a mapping from strings to strings anyways.